docs/mmteb/points/641.jsonl

```diff
@@ -0,0 +1 @@
+{"GitHub": "crystina-z", "New dataset": 2}
```

Comment:
I would remove the 2 points from adding a dataset and instead add 4 points to (bug)-fixes

Feel free to add a point for the dataset annotation as well

There is also a bonus for tel, ind, saw, yor

so that makes 4 + 1 + 4x4 = 21 points

Please also add the points for the reviewers.